### ISSUE TITLE
Need tangent

### ISSUE THREAD

**Issue by dimonbavly** [CadQuery/cadquery#165]
Is there a function to draw a tangent to an object (for example, a tangent to a circle)? Also, for example, you need two touching circle.

**Comment by jmwright** [comment]
I don't think so. One area where CadQuery is lacking is in 2D operations. That has been discussed previously, and improvement of 2D operations is on the roadmap.

You could access the lower level PythonOCC API to do this, but I don't think there's anything currently in CadQuery's fluent API to do it.
